cpu-bind=MASK - bp1-compute016, task  0  0 [3920888]: mask 0xfffc000fffc0 set
Traceback (most recent call last):
  File "/user/home/zj22662/photon-net/calc_test2.py", line 84, in <module>
    run_evaluation(hard_predict=False)
  File "/user/home/zj22662/photon-net/calc_test2.py", line 79, in run_evaluation
    iterate_models(output_folder, results_subfolder, model_number, current_conf, hard_predict)
  File "/user/home/zj22662/photon-net/calc_test2.py", line 58, in iterate_models
    utils.evaluate_and_save_test_loss(
  File "/user/home/zj22662/photon-net/p_pack/utils.py", line 271, in evaluate_and_save_test_loss
    acc, l_val = _single_metrics(input_config)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/user/home/zj22662/photon-net/p_pack/utils.py", line 226, in _single_metrics
    _, class_probs, n_p, _ = model.predict_reupload(
                             ^^^^^^^^^^^^^^^^^^^^^^^
ValueError: Non-hashable static arguments are not supported. An error occurred while trying to hash an object of type <class 'numpy.ndarray'>, [1 0 0 0 0 0 0 0 0 0]. The error was:
TypeError: unhashable type: 'numpy.ndarray'

